### DISCUSSION TITLE
Is this thing I notice in Cinnamon a bug or feature?

### DISCUSSION THREAD

**Discussion by SuperSentient** [keepassxreboot/keepassxc#8973] [Q&A] [answered]
Whenever I auto type on Linux Mint Cinnamon KeepassXC minimizes its window from full screen to a shrunken one.
Is this a bug or a feature?

**Comment by droidmonkey** [comment, marked as answer]
Could be a bug in cinnamon that controls window sizing. When you Auto-Type, the window will be minimized to the tray, but it shouldn't lose its maximized flag (ie taking the full screen size).

**Reply by SuperSentient** [reply]
> Could be a bug in cinnamon that controls window sizing. When you Auto-Type, the window will be minimized to the tray, but it shouldn't lose its maximized flag (ie taking the full screen size).

Sorry for the late reply. But could this harm my KDBX file at all?

**Reply by droidmonkey** [reply]
No

**Reply by SuperSentient** [reply]
Thanks. :+1:  :)
